src/components/Fields/TextArea.test.js

```diff
@@ -0,0 +1,253 @@
+/* global describe, expect, it, shallow, mount */
+import React from 'react';
+import sinon from 'sinon';
+import TextArea from './TextArea';
+import Tooltip from '../Tooltip/Tooltip';
+import IconRequired from '../Icon/IconRequired';
+
+function createTextarea() {
+    return shallow(<TextArea label="My Input" />);
+}
+
+describe('TextArea', () => {
+    it('renders an div wrapper element', () => {
```

Comment:
Could you please use the new `isConformant` helper in this file to ensure correct classes, etc.?